spec/requests/form_odk_rendering_spec.rb

```diff
@@ -188,6 +194,28 @@
       do_request_and_expect_success
       expect(tidyxml(response.body)).to eq prepare_odk_expectation("repeat_group_form.xml", form)
     end
+  end
+
+  context "with multiple conditions and skip rules" do
+    let(:form) do
+      create(:form, question_types: [
+        "integer",
+        "integer",
+        "integer",
+        ["integer", "integer", "integer"],
+        "integer",
+        ["integer", "integer", ["integer", "integer"]],
+        "integer"
+      ])
+    end
+
+    it "should render proper xml" do
+      form.c[0].skip_rules.build(destination: "item", dest_item: form.c[6])
+    end
```

Comment:
What is this? Doesn't seem like it does anything.

Reply:
False start I lost track of and will remove. Actual spec is farther down the file.